markdown/dt-otel/dt-otel.md

```diff
@@ -111,12 +111,95 @@ The `.env` file contains several environment variables that are used within the
 * Open Telemetry contribution repository collector docker image tag and version `otel/opentelemetry-collector-contrib:0.67.0`
 * Solace PubSub+ broker docker image tag and version `solace/solace-pubsub-standard:10.2`
 
-## Configuring the PubSub+ Event Broker: Software
+## Three Options for Config Management
+Duration: 0:03:00
+
+You can (generally) configure the Solace PubSub+ broker using three different methods; this section provides a very brief outline of those.
+
+Each section of this CodeLab that performs a configuration step on the Solace broker will include all three options.
+
+Negative
+: **Note:** do _not_ perform all 3 configuration options, just choose 1 for each section.
+
+For any configuration management, you will need a username/password with either admin or read/write level privileges.
+
+### PubSub+ Manager GUI
```

Comment:
I would suggest numbering these titles
```
### 1. PubSub+ Manager GUI
### 2. SEMP Management API
### 3. Command Line Interface (CLI)
```

Reply:
just on this page or everywhere throughout the code lab?

Reply:
I would suggest just on this page